### DISCUSSION TITLE
Welcome to NFL_Kaggle Discussions!

### DISCUSSION THREAD

**Discussion by mrinal18** [Mrinal18/NFL_Kaggle#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrinal18** [comment]
@lalalaurenville @gsvsumanth @Archana-bc Please let me know if you guys are able to access all the Discussions and Kanban board in Project tab

**Reply by lalalaurenville** [reply]
👍🏽
